src/profile-logic/function-info.js

```diff
@@ -244,3 +244,30 @@ export function stripFunctionArguments(functionCall) {
   }
   return functionCall;
 }
+
+export function removeTemplateInformation(functionName) {
```

Comment:
I'm not sure about the repercussions of running JS functions through this. We live in a weird world of Rust, WASM, ASM, etc. However we're not correctly demangling these... yet. It might be worth only running this code on C++ for now, and then revisit it once we properly support WASM and ASM code.

Reply:
Oh and also our engine inserts `<` characters into JS function names. I think that might break this function. I forget the exact particulars of how they get constructed but I believe something like this should pass. 'expect(removeTemplateInformation('foo/<bar')).toEqual('foo/<bar');'

Reply:
Yeah I think this is done for Promises (or maybe all callbacks in general).